src/pytorch_lightning/strategies/ddp2.py

```diff
@@ -11,6 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import annotations
```

Comment:
```suggestion
```

We cannot use this as for some python versions it will convert all type annotations to strings which is currently breaking jsonargparse